test/external/nrntest/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-list(APPEND TEST_ENV PATH=${PROJECT_BINARY_DIR}/bin:${PROJECT_SOURCE_DIR}/external/nrntest:$ENV{PATH})
+list(APPEND TEST_ENV PATH=${PROJECT_BINARY_DIR}/bin:${CMAKE_INSTALL_PREFIX}/bin:${PROJECT_SOURCE_DIR}/external/nrntest:$ENV{PATH})
 add_test(external_nrntest ${CMAKE_COMMAND} -E env ${TEST_ENV} $ENV{SHELL} ${PROJECT_SOURCE_DIR}/external/tests/nrntest/runtests)
```

Comment:
Adding install prefix here is not desirable. We want test to use build directory artefact (imagine the scenario where install prefix has old build)

Reply:
Indeed, just testing